docgen/src/guides/calendar-widget.md

```diff
@@ -7,6 +7,7 @@ withHeadings: true
 navWeight: 20
 editable: true
 githubSource: docgen/src/guides/calendar-widget.md
+indexName: concert_events_instantsearchjs
```

Comment:
doesn't this need api key and app id too?

Reply:
I'm using the default `appId` and `apiKey` values here, only the index name is different. Should I specify all of them?

Reply:
ah, I thought it was on your own app; all good to go then!